### PR TITLE
amend fsanitize SANFLAG in makefile

### PR DESCRIPTION
**Pull request by abheekg**

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -5,7 +5,17 @@ LDFLAGS = -lm
...
+
+cc_asan:=$(shell expr `gcc -dumpversion` \>= 4.8)
+cc_ubsan:=$(shell expr `gcc -dumpversion` \>= 4.9)
+
+ifeq ($(cc_asan),1)
+    SANFLAGS += -fsanitize=address
+endif
+
+ifeq ($(cc_ubsan),1)
+    SANFLAGS += -fsanitize=undefined
```
Comment (maedoc) [inline makefile]: please add ` -fsanitize=float-cast-overflow` and then I'll test & merge

**Comment by abheekg** [comment]
@maedoc I intentionally didn't add it `-fsanitize=float-cast-overflow` . It is not working in both of my systems as mentioned earlier. Please let me know if it can be solved my tweaking with my installed packages.

**Comment by maedoc** [comment]
You're right, it isn't present in GCC 4.9.2, at least. GCC 5.3 supports it, but I agree, let's drop it for now. The address sanitizer is the most worthwhile one as it is much faster than valgrind.

**Comment by maedoc** [comment]
@abheekg :+1: 

(This was a small change so a PR from your master was OK, but generally, please PR from a topic branch)

**Pull request merged**